src/System.IO.Pipelines/System/IO/Pipelines/IPipeWriter.cs

```diff
@@ -1,20 +1,16 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.Buffers;
+using System.Threading;
+
 namespace System.IO.Pipelines
 {
     /// <summary>
     /// Defines a class that provides a pipeline to which data can be written.
     /// </summary>
-    public interface IPipeWriter
+    public interface IPipeWriter: IOutput
```

Comment:
I wonder if we should not rename the type to IAsyncOutput

Reply:
Which type? IPipeWriter?